ts/nni_manager/training_service/reusable/environments/remoteEnvironmentService.ts

```diff
@@ -208,6 +208,33 @@ export class RemoteEnvironmentService extends EnvironmentService {
         this.remoteMachineMetaOccupiedMap.set(remoteEnvironment.rmMachineMeta, false);
     }
 
+    private async getScript(environment: EnvironmentInformation): Promise<string> {
+        const executor = await this.getExecutor(environment.id);
+        const logLevel = getLogLevel();
+        let script: string = environment.command;
+
+        if (executor.getIsWindows()) {
+            const prepare = `envs\\install_nni.cmd && mkdir envs\\${environment.envId} && cd envs\\${environment.envId}`;
+            const startrun = `cd .. && install_nni.cmd && cd ${environment.envId} && python -m nni.tools.trial_tool.trial_runner`;
```

Comment:
seems install twice? `envs\\install_nni.cmd` and `install_nni.cmd`

and multi `cd` can be simplified.

Reply:
Fixed.